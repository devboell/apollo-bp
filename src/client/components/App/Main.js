import styled from 'styled-components'

const Main = styled.main`
  display: flex;
  justify-content: space-around;
`

export default Main
