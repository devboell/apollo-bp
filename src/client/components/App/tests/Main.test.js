import React from 'react'
import 'jest-styled-components'

import Main from '../Main'

it('Main, styled component snapshot', () => {
  const wrapper = shallow(<Main />)
  expect(wrapper).toMatchSnapshot()
})
