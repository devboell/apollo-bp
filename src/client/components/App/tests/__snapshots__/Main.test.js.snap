// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Main, styled component snapshot 1`] = `
.c0 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
}

<main
  className="c0"
/>
`;
